pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -53,6 +53,13 @@ type expandResponse struct {
 // they either explicitly do have a relationship or explicitly do not.
 type userRelationshipStatus int
 
+func (r *userRelationshipStatus) InvertRelationshipStatus() userRelationshipStatus {
```

Comment:
Why the pointer receiver instead of the non pointer receiver here? Since we just directly de-ref it, does it make more sense to just use a non-pointer?